akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -121,7 +123,7 @@ object FSM {
       extends NoSerializationVerificationNeeded {
     private var ref: Option[Cancellable] = _
     private val scheduler = context.system.scheduler
-    private implicit val executionContext = context.dispatcher
+    private implicit val executionContext: ExecutionContextExecutor = context.dispatcher
```

Comment:
ExecutionContext

Reply:
Adding type for implicit members is now performed by scalafix